solutions/swb-reference/src/templates/README.md

```diff
@@ -1,30 +1,48 @@
-# As we enable more hosting account templates, we will document the purpose of each here
+# Onboarding a hosting account template
 
-## onboard-account.cfn.yaml
-A CFN template that creates all resources SWB needs to manage the relationship between the main and hosting accounts.
+Here are a few onboarding template choices that you could use for onboarding your hosting account. The difference between each of these templates is based on the level of network flexibility you need. This is where your SWB compute resources would be hosted.
 
-## onboard-account-byon.cfn.yaml
-A CFN template that does everything onboard-account.cfn.yaml does, except allows the hosting account to use its own VPCs.
+----
 
-## The following are CFN templates that help setting up networks for transit gate ways in the BYON scenario
-### new-firewall-vpc-TGW.yaml
-A template to help a Network account set up VPCs, a firewall, and a Transit Gateway. 
+## Onboard Account
```

Comment:
Might want to add `Default` somewhere in here so its easy to refer back to when we're talking about other types of onboarding methods.